backend/corpora/dataset_processing/process.py

```diff
@@ -225,7 +226,7 @@ def cancel_dataset(dataset_id):
         logger.info("Upload Canceled.")
 
 
-def update_db(dataset_id, metadata=None, processing_status=None):
+def update_db(dataset_id, metadata: dict = None, processing_status: dict = None):
```

Comment:
```suggestion
def update_db(dataset_id: str = None, metadata: dict = None, processing_status: dict = None):
```

Reply:
I don't think we want to have a default value of None, but the type hint would be fine